target-platform/p2-repo-common/pom.xml

```diff
@@ -172,6 +172,11 @@
 									<version>${org.apache.commons.fileupload.version}</version>
 								</artifactItem>
 								<artifactItem>
+								    <groupId>org.apache.commons</groupId>
+								    <artifactId>commons-exec</artifactId>
+								    <version>${commons-exec.version}</version>
```

Comment:
I would have org.apache.commons.exec.version instead or commons-exec.version. In the configuration file I would keep org.apache.commons.exec.version=1.3